tools/feature_support_matrix.rb

```diff
@@ -0,0 +1,68 @@
+require 'csv'
+
+def usage
+  <<-USAGE
+    Usage:
+      `bin/rails r tools/feature_support_matrix.rb`
```

Comment:
Suggest putting this at the top so you can invoke this with plain ruby:
`require File.expand_path('../config/environment', __dir__)`

This will load rails in a way that's nearly 100% identical was bin/rails runner.  Runner is for inline ruby strings, not actual files.   See also #4307.

the side effect is you don't have to deal with rails runner intercepting any ARGs you pass to the script.